lib/gradient.coffee

```diff
@@ -2,17 +2,22 @@ class PDFGradient
   constructor: (@doc) ->
     @stops = []
     @embedded = no
-    @transform = [1, 0, 0, 1, 0, 0]
+    @_transform = [1, 0, 0, 1, 0, 0]
     @_colorSpace = 'DeviceRGB'
     
   stop: (pos, color, opacity = 1) ->
     opacity = Math.max(0, Math.min(1, opacity))
     @stops.push [pos, @doc._normalizeColor(color), opacity]
     return this
     
-  embed: ->
-    return if @embedded or @stops.length is 0
+  transform: (m11, m12, m21, m22, dx, dy) ->
+    @_transform = [m11, m12, m21, m22, dx, dy]
+    return this
```

Comment:
I just renamed the function and variable to preserve compatibility with the old api